src/components/PressableWithSecondaryInteraction/index.js

```diff
@@ -26,6 +27,16 @@ class PressableWithSecondaryInteraction extends Component {
         this.pressableRef.removeEventListener('contextmenu', this.executeSecondaryInteractionOnContextMenu);
     }
 
+    executeSecondaryInteraction(e) {
```

Comment:
This is missing [jsdocs](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)

```
    /**
     * @param {Event} e
     */
```